ui.content/src/main/content/jcr_root/conf/acm/settings/snippet/available/core/argument/path.yml

```diff
@@ -0,0 +1,15 @@
+group: argument
+name: argument_path
+content: |
+  args.path("${1:name}") { label = "${2:label}"; root = "${4:value}" }
+documentation: |
+  An argument that represents a repository path with a path picker dialog.
+  
+  The path picker allows users to browse and select paths from the JCR repository.
+
+For example:
+  ```groovy
+  args.path("contentPath") { label = "Content Path" }
```

Comment:
root = "/content" :)

Reply:
I would leave as it is to show that this param is not obligatory :)